### DISCUSSION TITLE
inverse tone mapping doesn't work for subtitles

### DISCUSSION THREAD

**Discussion by rainlycoris** [mpv-player/mpv#14614] [Q&A]
Even blend-subtitles = yes, the subtitles are very dark. Is there any way to solve them?

```
vo = gpu-next
gpu-context = macvk
hwdec = no

scale = ewa_hanning
scale-antiring = 0.5
cscale = catmull_rom
dscale = spline36
dither-depth = 10
correct-downscaling = yes
linear-downscaling = yes
sigmoid-upscaling = yes
hdr-compute-peak = yes
hdr-peak-percentile = 99.995
hdr-contrast-recovery = 0.30
allow-delayed-peak-detect = no
deband = yes
fbo-format = rgba32f
icc-profile-auto = yes

inverse-tone-mapping = yes 
tone-mapping = bt.2446a
target-colorspace-hint = yes
target-trc = pq
target-prim = bt.2020
target-peak = 1000

blend-subtitles = yes
sub-auto = fuzzy
fullscreen = yes

hidpi-window-scale = no
```

<img width="1512" alt="截屏2024-07-30 22 40 53" src="https://github.com/user-attachments/assets/4a6afc7d-41e9-4020-8842-e0201d44820a">

The screenshots of the hdr content are not very good, but you can still notice the captions which in the top left and bottom right are too dark.

Thanks.

**Comment by kasper93** [comment]
This will probably help https://code.videolan.org/videolan/libplacebo/-/merge_requests/630

**Reply by rainlycoris** [reply]
Thank you very much! But I don't know anything about it, so what can I do, do I need to recompile mpv or other?

**Reply by llyyr** [reply]
Wait for it to be merged or compile mpv locally with that patch applied
